snuba/settings_base.py

```diff
@@ -9,9 +9,45 @@
 PORT = 1218
 
 DEFAULT_DATASET_NAME = 'events'
-DISABLED_DATASETS = {}
 DATASET_MODE = 'local'
 
+DISABLED_DATASETS = {}
+
+KAFKA_CLUSTERS = {
+    "default": {
+        "brokers": ['localhost:9093'],
+        "max_batch_size": 50000,
+        "max_batch_time_ms": 2 * 1000,
+        "queued_max_message_kbytes": 50000,
+        "queued_min_messages": 20000,
+    }
+}
+
+DATASETS = {
+    "events": {
+        "consumer": {
+            "kafka_cluster_base": "default",
```

Comment:
I referred to the cluster by key instead of referring directly to the entry in KAFKA_CLUSTER ("kafka_cluster": KAFKA_CLUSTER['default']) . to make things easier when overriding parameters per environment. So test_docker can override a field in KAFKA_CLUSTER and that will be automatically taken into account in all datasets. So the structure of the environment files stays consistent with the one of settings_base.